src/pluginLoader.js

```diff
@@ -0,0 +1,660 @@
+/*
+ * This is a pared down version of hawtio-core in order to remove the outdated
+ * dependency as part the migration from bower to yarn.
+ *
+ * TODO Figure out a better strategy for API discovery so that this can be
+ * removed altogether.
+ *
+ */
+
+// log initialization
+/* globals Logger window console document localStorage $ angular jQuery navigator Jolokia */
+(function() {
+  'use strict';
+
+  Logger.setLevel(Logger.INFO);
+  Logger.storagePrefix = 'hawtio';
+
+  Logger.oldGet = Logger.get;
+  Logger.loggers = {};
+  Logger.get = function(name) {
+    var answer = Logger.oldGet(name);
+    Logger.loggers[name] = answer;
+    return answer;
+  };
```

Comment:
I think we can remove lines 18-24. I don't see where `Logger.loggers` is ever used.